easybuild/tools/containers.py

```diff
@@ -245,7 +245,9 @@ def build_singularity_image(def_path):
     # resolve full path to 'singularity' binary, since it may not be available via $PATH under sudo...
     singularity = which('singularity')
 
-    cmd = "sudo %s build %s %s %s" % (singularity, cmd_opts, img_path, def_path)
+    singularity_tmp = build_option('container_tmp_path')
```

Comment:
@migueldiascosta What if this is not specified, you'll inject `SINGULARITY_TMPDIR=None` into the command?

Fixes in https://github.com/migueldiascosta/easybuild-framework/pull/1

Reply:
@boegel you're right, I assumed it would be empty and singularity would default to `/tmp` (and thought I had tested it but apparently hadn't)